drivers/i2s/i2s_stm32_sai.c

```diff
@@ -849,4 +860,4 @@ static DEVICE_API(i2s, i2s_stm32_driver_api) = {
 	K_MSGQ_DEFINE(queue_##index, sizeof(struct queue_item), CONFIG_I2S_STM32_SAI_BLOCK_COUNT,  \
 		      4);
 
-DT_INST_FOREACH_STATUS_OKAY(I2S_STM32_SAI_INIT)
+DT_INST_FOREACH_STATUS_OKAY(I2S_STM32_SAI_INIT)
```

Comment:
Please discard this change straight from commit *"drivers: i2s: i2s_stm32_sai add support for stm32l4xx"*.